test/Neo4jClient.Extension.UnitTest/Cypher/FluentConfigMergeTests.cs

```diff
@@ -33,24 +33,24 @@ public void OneDeep()
             var text = q.GetFormattedDebugText();
             Console.WriteLine(text);
 
-            Assert.AreEqual(@"MERGE (person:SecretAgent {id:{
-  id: 7
-}.id})
-ON MATCH SET person.spendingAuthorisation = 100.23
```

Comment:
I'm using Neo4jClient and Neo4jClient.Extensions. When using Neo4jClient and the `Where` extension method, I noticed that Neo4jClient was not using camelCase. Which make both library incompatible on the same database.
As this is something that cannot be easily changed in Neo4jClient, I decided to unify the behavior and use the ContractResolver of Neo4jClient in Neo4jClient.Extensions

Reply:
Ah, hmmm - well, I had to double to check, since I actually added camelCase support to the Neo4jClient library https://github.com/DotNet4Neo4j/Neo4jClient/pull/66/files - and while the default may not be that, it seems like an unnecessary change to the tests in this library... It'd be great if this change could limit it's self to the upgrade to netstandard2 which I absolutely agree is long over-due. Thanks